app/views/service_calendars/master_calendar/pppv/template/_select_column.html.haml

```diff
@@ -0,0 +1,30 @@
+-# Copyright © 2011 MUSC Foundation for Research Development
```

Comment:
Replaced `Dashboard::ServiceCalendars#select_column` lib file method so that we could rerender it